src/app/rpmostree-builtin-db.cxx

```diff
@@ -61,7 +61,7 @@ rpmostree_db_option_context_parse (GOptionContext *context,
                                        argc, argv,
                                        invocation,
                                        cancellable,
-                                       NULL, NULL, NULL, NULL,
```

Comment:
This `NULL NULL NULL NULL` thing screams bad API design too, so it's good to reduce it!